src/server/scripts/Kalimdor/TempleOfAhnQiraj/boss_cthun.cpp

```diff
@@ -585,6 +608,27 @@ struct boss_cthun : public BossAI
 
         //Body Phase
         uint8 _fleshTentaclesKilled;
+
+        //If no-one is outside
+        bool _stomachDeathScheduled;
+
+        //Map variable
+        Map* map;
+
+        bool IsEveryoneAffectedByDigestiveAcid() const
```

Comment:
```suggestion
        bool _IsEveryoneAffectedByDigestiveAcid() const
```
But it's up to you to be honest. But does make it clear that this function is private